### DISCUSSION TITLE
Observation = None and done = True when using from_file method

### DISCUSSION THREAD

**Discussion by rezabayani** [ds4dm/ecole#353] [General]
Hi,

My Ecole (v8) is having trouble reading observations from the optimization model stored as mps file. Basically, it returns None observation and True done flag when I reset the environment. I did the same approach with Ecole v7 and it was okay back then. Interestingly, when I tried feeding other mps files found online, it is okay with some of them and not with others. I have attached 2 images. Number 1 is when Ecole is acting as intended, using an mps model found online. Number 2 is when Ecole is acting funny, with an mps file I generated myself and was okay with Ecole v7.

![ecole_incorrect](https://user-images.githubusercontent.com/59495621/222301757-a2e07fef-ac5c-43f9-ab73-f964d07ee240.PNG)
![ecole_correct](https://user-images.githubusercontent.com/59495621/222301762-fa51b5df-882d-4fca-b412-6b282f00a2ab.PNG)

**Comment by xueno6** [comment]
Nice post, I found the same issue, did you find any solution?
